### PR TITLE
course data: Create an eCommerce Store with Next.js and Stripe Checkout

### PR DESCRIPTION
**Pull request by zacjones93**
adds freshness review, and related content to course landing page

CleanShot 2021-02-15 at 20 10 34@2x

### CODE REVIEW

File: src/components/layouts/collection-page-layout.tsx [hunk 1]
```diff
@@ -483,8 +483,8 @@ const CollectionPageLayout: React.FunctionComponent<CoursePageLayoutProps> = ({
                 <CoursePodcast podcast={podcast} instructorName={full_name} />
               )}
               <div
-                className={`grid md:gap-x-5 ${
-                  prerequisites && topics ? 'grid-cols-2' : 'grid-cols-1'
+                className={`grid grid-cols-1 md:gap-x-5 ${
```
Comment (zacjones93) [inline src/components/layouts/collection-page-layout.tsx:486]: we weren't collapsing columns in mobile view

File: src/data/courseDependenciesData.js [hunk 2]
```diff
@@ -929,7 +929,7 @@ const courseDependencyData = (courseSlug) =>
         slug: 'create-an-ecommerce-store-with-next-js-and-stripe-checkout-562c',
         dependencies: {
           react: '>= 17.0.1',
-          next: '>=9.5.5',
+          next: '^10.0.6',
```
Comment (laurosilvacom) [inline src/data/courseDependenciesData.js:932]: Colby updated all the lesson to ^10.0.6 - [reference](https://github.com/colbyfayock/space-jelly-store-workshop/commit/40ef796469c741e6ea9744a9dc92f7cfe9e9282f)

File: src/data/courseDependenciesData.js [hunk 3]
```diff
@@ -959,61 +959,34 @@ const courseDependencyData = (courseSlug) =>
             slug: 'the-beginner-s-guide-to-react',
           },
         ],
-        courseProject: {
```
Comment (laurosilvacom) [inline src/data/courseDependenciesData.js]: Adding a portfolio project block that points to the custom landing page is confusing. I think both now both course pages can exist without linking to each other.

File: src/data/courseDependenciesData.js [hunk 4]
```diff
@@ -959,61 +959,34 @@ const courseDependencyData = (courseSlug) =>
...
-          label: 'Portfolio Project',
-          url:
-            '/projects/create-an-ecommerce-store-with-next-js-and-stripe-checkout',
-          text: `After this course, you'll be ready to start building your own e-commerce store. Need an idea? Start with this Project!
-      `,
-        },
         pairWithResources: [
+          {
```
Comment (laurosilvacom) [inline src/data/courseDependenciesData.js:963]: Scope the  `pairWithResources` to content that was specifically designed for this course.

**Review by laurosilvacom — APPROVED** [review]

**Pull request merged**